ouroboros-consensus/src/Ouroboros/Consensus/Storage/ImmutableDB/Impl/Types.hs

```diff
@@ -111,7 +112,7 @@ data TraceEvent blk =
     NoValidLastLocation
   | ValidatedLastLocation ChunkNo (Tip blk)
     -- Validation of previous DB
-  | ValidatingChunk  ChunkNo
+  | ChunkValidationEvent (TraceChunkValidation ChunkNo)
```

Comment:
Why not adding here the `validateTo ~ ChunkNo` value? (instead of in `TraceChunkValidation`?)

Reply:
In case we want to add different types of events in `TraceChunkValidation` that do not use `ChunkNo`.

TODO: add other chunk related events to `TraceChunkValidation`.

Reply:
Done